src/eval-frame/index.jsx

```diff
@@ -20,6 +20,7 @@ import messagePasserEval from "../shared/utils/redux-to-port-message-passer";
 import { iodide } from "./iodide-api/api";
 
 import "./port-to-editor";
+import { initializeDefaultKeybindings } from "../editor/initialization/keybindings";
```

Comment:
can you move the relevant code for intercepting the backspace to a new file in `src/shared` and import that code here and in `src/editor`?

the reason for doing so is that importing `../editor/initialization/keybindings` into the eval-frame causes the import of
`import tasks from "../user-tasks/task-definitions"` from the editor side, which imports `actions` from the editor side, etc. in this ase, it would just import some extra code i think, but we try to maintain import hygiene by putting code that will be run in both the editor and eval-frame into `src/shared` because in some cases importing things in both sides of the app can cause conflicts and weird bugs.

thanks @Algentile ! :-)